index.js

```diff
@@ -191,14 +191,42 @@ function byName (name, context) {
   name = browserslist.aliases[name] || name
   if (context.mobileToDesktop &&
     browserslist.desktopNames[name]) {
-    return replaceDataName(
-      browserslist.data[browserslist.desktopNames[name]],
-      name
-    )
+    var desktopData =
+      browserslist.data[browserslist.desktopNames[name]]
+    if (name === 'android') {
+      return normalizeAndroidData(cloneData(
+        browserslist.data[name]
+      ), desktopData)
+    } else {
+      var cloned = cloneData(desktopData)
+      cloned.name = name
+      return cloned
+    }
   }
   return browserslist.data[name]
 }
 
+function normalizeAndroidVersions (androidVersions, chromeVersions) {
+  var firstEvergreen = 37
```

Comment:
If we use this bar in multiple places, we need to move it to `var ANDROID_FIRST_EVERGREEN` constant.